### PR TITLE
Javascript Card Game

### PR DESCRIPTION
**Pull request by Puvikaran**
Still card values to be sort and sum

### CODE REVIEW

File: Puvikaran/cardgame.js [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+    function Deck() {
+        this.newDeck = makeDeck();
+        this.shuffleDeck = this.deck; // Shuffle(deck);
+        this.addCard = addCard();
+        this.removeCard = removeCard();
+        //this.cardCount = deckCardCount;
+
+        //console.log("deck function");
+
+        function Card(values, suites) {
```
Comment (giju) [inline Puvikaran/cardgame.js]: externalise this class and add associated functions to the class.

File: Puvikaran/cardgame.js [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+                    // console.log("card: " + newcard.name);
+                    deck.push(newcard);
+
+                }
+            }
+            // console.log("CURRENT DECK: ", deck);
+            return deck;
+        }
+
+        function Shuffle() {
```
Comment (giju) [inline Puvikaran/cardgame.js]: make this a method of the deck class

File: Puvikaran/cardgame.js [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+        function Card(values, suites) {
+            this.values = values;
+            this.suites = suites;
+            this.name = suites + '' + values;
+
+            // console.log(this.name + '\t');
+            document.write(this.name + "\t");
+        }
+
+        function makeDeck() {
```
Comment (giju) [inline Puvikaran/cardgame.js:21]: make this the constructor of deck

File: Puvikaran/cardgame.js [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+            }
+            console.log(deck);
+            return this;
+        }
+
+        function addCard(deck) {
+            this.playingCards.push(deck);
+        }
+
+        function removeCard() {
```
Comment (giju) [inline Puvikaran/cardgame.js:55]: move these functions to deck / player

**Review by giju — COMMENTED** [review]
let me know once the changes are made

**Review by giju — CHANGES_REQUESTED** [review]
move all classes to separate files , Once you have made the changes let me know

**Comment by Puvikaran** [comment]
Card values calculated and Players sorted based on card values,
Files Separated and the remaining changes mentioned needs to be done 
cardgame

**Pull request merged**